lib/mpl_toolkits/mplot3d/axes3d.py

```diff
@@ -345,7 +345,7 @@ def _equal_aspect_axis_indices(self, aspect):
             ax_indices = [1, 2]
         return ax_indices
 
-    def set_box_aspect(self, aspect, *, zoom=1):
+    def set_box_aspect(self, aspect, *, zoom: float = 1):
```

Comment:
```suggestion
    def set_box_aspect(self, aspect, *, zoom=1.0):
```

This is a smaller change that should address the inferred type without broaching adding typehints in an unhinted module. 

Again, if you are expecting type hints with 3D you _will_ be continually disappointed at present, so caution is in order in general.

I'm still skeptical of the whack-a-mole here, but this would be my preferred short term solution to this particular typing error.

Reply:
I do agree that changing the type for a single line is a bit ridiculous, especially because the same Problem occurs at least a few dozen times in this file. 

After looking at the file again there are only about 100 function, many without arguments, so wouldn't take too long to add typing to the entire file. So maybe I'll do that later.